game/CMakeLists.txt

```diff
@@ -9,6 +9,8 @@ project ("game")
 set(CMAKE_CXX_STANDARD 20)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
+option(COPY_GAME_ASSETS "Copy assets to build directory" ON) # when prototyping, set to OFF
```

Comment:
turn off if you are running on a slower machine or don't want to kill your drive